satpy/etc/composites/fci.yaml

```diff
@@ -64,3 +64,34 @@ composites:
       - name: ndvi_hybrid_green_raw
       - name: vis_04
     standard_name: true_color_raw
+
+  true_color_reproduction:
```

Comment:
I don't see a matrix for fci, will it still work?

Reply:
Not added it yet, will do soon.